config/ranks/military.txt

```diff
@@ -1,9 +1,9 @@
 Captain=CPT
-Executive Officer=LT
-Head of Security=WO
-Chief Engineer=WO
-Research Director=WO
-Chief Medical Officer=WO
+Executive Officer=CDR
```

Comment:
Honestly I'd at least bring everything down by one and make the captain a commander and the exec a lieutenant commander